moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -73,7 +73,17 @@ const makeAdvancedSearchWhereFilters = (filters) =>
 
           // If it is a number or boolean, it does not need quotation marks
           // Otherwise, add quotation marks for the query to identify as string
-          value = type in ["number", "boolean"] ? value : `"${value}"`;
+          if (["number", "boolean"].includes(type)) {
+            // eslint-disable-next-line no-self-assign
+            value = value;
+          } else if (type === "array") {
```

Comment:
This construct here is to cause non-ops for these two types of `type.` It's not pretty, made obvious by having to disable that linting line, but .. a switch statement would have to start with switch(true) and then put expressions in the cases -- hardly any more elegant. This at least felt very explicit about what's happening, perhaps even to a fault.

Reply:
Goodness, that is remarkably better. Thank you!